src/settings/user.rs

```diff
@@ -71,16 +65,8 @@ impl UserConfiguration {
     ) -> Self {
         Self {
             dummy_variable_rgx: configuration.dummy_variable_rgx,
-            exclude: configuration
-                .exclude
-                .into_iter()
-                .map(Exclusion::from_file_pattern)
-                .collect(),
-            extend_exclude: configuration
-                .extend_exclude
-                .into_iter()
-                .map(Exclusion::from_file_pattern)
-                .collect(),
+            exclude: configuration.exclude,
+            extend_exclude: configuration.extend_exclude,
```

Comment:
I think we need to map these to strings somehow -- the current output of `cargo run resources/test/fixtures/ --show-settings` isn't very user-friendly:

![Screen Shot 2022-11-22 at 10 10 44 PM](https://user-images.githubusercontent.com/1309177/203462366-43255dde-4a8b-4b79-8fcb-4871486d711f.png)

Reply:
Sounds good! Thank you for your contribution :)

If at any point you feel like you won't have time to get to the comments, just let me know and we can figure out a plan.

Reply:
e4d537d and 9e59346 - Had to keep an uncompiled version of the globs to get the strings needed but settings now look much cleaner